src/Ingredients/DisplayRecipes.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+
+namespace BreadAndIfit\Ingredients;
+
+
+class DisplayRecipes
+{
+    /**
+     *takes the JSON data returned from the API request and returns a string with HTML markup
+     *
+     * @param $json API response in JSON format
+     * @return string relevant recipes based on user selected ingredients or a error message string
+     */
+    public static function outputRecipes($json)
+    {
+        $recipes = json_decode($json)->results;
+
+        $validate = self::validateRecipe($recipes);
+
+        if ($validate) {
+            return self::outputHTML($recipes);
+        } else {
+            return '<div id="mainPannel">
+                    <div class="row recipe mx-auto">
+                        <div class="col-10 col-lg-8">
+                            <h5 class="card-title">Invalid response received from puppy API</h5>                   
```

Comment:
not sure puppy API makes any sense as a response to the user?

Reply:
We've redirected to a different error message 'There are no recipes available which use the ingredient(s) you selected'